clap_derive/src/derives/attrs.rs

```diff
@@ -104,10 +104,7 @@ pub struct Attrs {
 /// The output of a generation method is not only the stream of new tokens but also the attribute
 /// information of the current element. These attribute information may contain valuable information
 /// for any kind of child arguments.
-pub struct GenOutput {
-    pub tokens: proc_macro2::TokenStream,
-    pub attrs: Attrs,
-}
+pub type GenOutput = (proc_macro2::TokenStream, Attrs);
```

Comment:
I don't think this should have been made a tuple. Reduces the readability of the code. Instead seeing `.tokens` and understanding, I now see `.0`

Reply:
How do you like this snippet?

https://github.com/clap-rs/clap/blob/20cdc8384f9830bbca7bfcbf4cd2ffdfc96e4eca/clap_derive/src/derives/clap.rs#L401-L406

This is what `struct` leads to.

Stuff like this is present in several places whereas `.0` occurs just once. I think the tuple is the lesser of two evils. It's subject of the upcoming refactoring anyway, so...